doc/source/serve/advanced-guides/grpc-guide.md

```diff
@@ -0,0 +1,247 @@
+(serve-set-up-grpc-service)=
+# Set Up gRPC Service
+
+This section helps you understand how to:
+- build a user defined gRPC service and protobuf
+- start Serve with gRPC proxy
+- deploy gRPC applications
+- send gRPC requests to Serve deployments
+- health checks proxies
+- gRPC request metadata 
+- more examples
+- error handling
```

Comment:
each of these bullets should start with a verb (like "deploy gRPC applications"). these two and the two above don't.